commands/system-prune.ts

```diff
@@ -0,0 +1,9 @@
+import vscode = require('vscode');
+
+export function systemPrune() {
+
+    let terminal = vscode.window.createTerminal("docker system prune");
```

Comment:
Yep I totally agree. That's why I didn't feel too strongly and wanted to check my thinking on it with your thoughts. Thanks dude!